llvm/lib/Target/RISCV/RISCVFoldMasks.cpp

```diff
@@ -18,6 +18,7 @@
 
 #include "RISCV.h"
 #include "RISCVSubtarget.h"
+#include "RISCVISelDAGToDAG.h"
```

Comment:
Alphabetize? Though I'm surprised a SelectionDAG header is needed in an MIR pass.

Reply:
It's needed for the masked pseudos table, which after everything else is moved over can also be moved into RISCVFoldMasks